app/src/main/java/com/example/taskmaster/taskFragment.java

```diff
@@ -0,0 +1,110 @@
+package com.example.taskmaster;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link taskFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link taskFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class taskFragment extends Fragment {

```

Comment:
We definitely want to change to an uppercase 'T', for class names, refactor to: `TaskFragment`